### PR TITLE
tickets/DM 47510

### PR DESCRIPTION
**Pull request by couger01**
- **Cycle 39 revision 8**
- **Update mthexapod startup script to use configuration from the obs-env if it is available.**
- **Update laser tracker startup script to use configuration from the obs-env if it is available.**
- **Update MTPtg configuration to increase az limits to +/- 260.**
- **Fix startup.sh in audiotrigger**
- **Remove rubintv from the conda packages list.**
- **Add EAS CSC (not to be confused with EAS collector)**
- **Update pointing model.**
- **Change trigger time for Sciplat build.**
- **Update version history**

### CODE REVIEW

File: doc/version-history.rst [hunk 1]
```diff
@@ -8,6 +8,42 @@ Version History
...
+  * rubin_scheduler
+
+* Updated mthexapod startup to use configuration from obs-env if available.
+* Updated laser tracker startup script to use configuration obs-env if available.
+* Updated MTPtg confeiguration to increase azimuth limits to +/- 260.
+* Fixed startup.sh in audiotrigger.
+* Removed rubintv from the conda packages list.
+* Added EAS CSC.
+* Updated pointing model.
+* Changed trigger time for sciplat build.
```
[the remark this answers is not in the document]
Comment (couger01) [reply doc/version-history.rst:45]: A change log represents something that has been done and should be written in the past tense. A git commit log is a representation of something that will be done and so is written with the future present tense.

[the remark this answers is not in the document]
Comment (sebastian-aranda) [reply doc/version-history.rst:45]: Thanks for that heads up :muscle:, I didn't have that definition in my head.

[the remark this answers is not in the document]
Comment (wvreeven) [reply doc/version-history.rst:45]: Sebastian is right. We write the version history of ts_cycle_build in present tense.

[the remark this answers is not in the document]
Comment (couger01) [reply doc/version-history.rst:45]: With standup today, we are now going to use the past tense starting with this revision.

**Pull request merged**